app/src/test/java/org/pesmypetcare/mypetcare/services/StubCommunityService.java

```diff
@@ -0,0 +1,263 @@
+package org.pesmypetcare.mypetcare.services;
+
+import org.pesmypetcare.mypetcare.features.community.forums.Forum;
+import org.pesmypetcare.mypetcare.features.community.forums.ForumNotFoundException;
+import org.pesmypetcare.mypetcare.features.community.forums.NotForumOwnerException;
+import org.pesmypetcare.mypetcare.features.community.groups.Group;
+import org.pesmypetcare.mypetcare.features.community.groups.GroupAlreadyExistingException;
+import org.pesmypetcare.mypetcare.features.community.groups.GroupNotFoundException;
+import org.pesmypetcare.mypetcare.features.community.posts.Post;
+import org.pesmypetcare.mypetcare.features.community.posts.PostAlreadyExistingException;
+import org.pesmypetcare.mypetcare.features.community.posts.PostNotFoundException;
+import org.pesmypetcare.mypetcare.features.users.User;
+import org.pesmypetcare.usermanager.datacontainers.DateTime;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.SortedSet;
+import java.util.TreeSet;
+
+public class StubCommunityService implements CommunityService {
+    private static final int HUSKY = 0;
+    private static final int TURTLES = 1;
+    private static final int ELEPHANTS = 2;
+    private static final int DINOSAURS = 3;
+
+    private static List<Group> groups;
+
+    static {
+        addStubDefaultData();
+    }
+
+    /**
+     * Add the default data for the stub.
+     */
+    public static void addStubDefaultData() {
+        addGroups();
+        addTags();
+        addFroums();
+    }
+
+    /**
+     * Add the forums.
+     */
+    private static void addFroums() {
+        new Forum("Washing", "John Doe", DateTime.Builder.buildFullString("2020-04-22T10:00:00"),
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String "2020-04-22T10:00:00" appears 2 times in the file.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5421176)